### ISSUE TITLE
Having to clear cache and history is disruptive to flow of work

### ISSUE THREAD

**Issue by astridu** [LD4P/sinopia_editor#2180]
The need to clear cache and history before starting each Sinopia session (and sometimes more than once, depending on how long you’ve been working) interrupts the workflow and has a major impact on usability.

Is there a way so that the user does not have to do this?

**Comment by jimfhahn** [comment]
Additional details from the UPenn Sinopia group: we have noticed that if a profile or some other javascript is older/cached then updated in the system doesn't work as expected because the older cached files are causing errors. I don't know if this is feasible in your current javascript framework, but a ?version=1 on javascript (or css) for the profiles when the javascript is minified may be one way to address. I believe this was possible in gulp ( https://www.npmjs.com/package/gulp-version-number ), but may vary depending on how you build. Happy to follow up with our group if there are questions.

**Comment by michelleif** [comment]
@astridu and @jimfhahn do you have examples of circumstances that require clearing of cache? are we talking about:
--not seeing new versions of templates? (this I thought was fixed see LD4P/sinopia_editor#1462)
--needing to re login even though it looks like you are still logged in? (see LD4P/sinopia_editor#556)

**Comment by jimfhahn** [comment]
I learned of this from a colleague who had attended the Profiles Affinity Group Meeting on April 15. In the group notes they suggest clearing cache and history. Those notes reference slack conversations. One example is here: https://ld4.slack.com/archives/CD4098H4Y/p1586892753102200

At the end of March, we had an issue where copying Works from another library into Sinopia Stage would not work unless cache was cleared, while Sinopia Production did work without issue.

**Comment by astridu** [comment]
IDK of a specific example, but U Minnesota also reported this as a problem.

**Comment by michelleif** [comment]
There _are_ situations where hard-refresh is required and these are documented in release notes. Closing this and if we have a specific example in future we will open a new ticket with that.
